pages/api/hello.ts

```diff
@@ -1,13 +1,13 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
-import type { NextApiRequest, NextApiResponse } from 'next'
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
we might wanna remove this into another PR

Reply:
Totally, I will deal with the API in my next task.